tests/storage/datavolume.go

```diff
@@ -391,8 +391,10 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 
 	Describe("[rfe_id:3188][crit:high][vendor:cnv-qe@redhat.com][level:system] Starting a VirtualMachine with an invalid DataVolume", func() {
 		Context("using DataVolume with invalid URL", func() {
-			It("shold be possible to stop VM if datavolume is crashing", func() {
+			It("should be possible to stop VM if datavolume is crashing", func() {
 				dataVolume := libstorage.NewDataVolumeWithRegistryImport(InvalidDataVolumeUrl, util.NamespaceTestDefault, k8sv1.ReadWriteOnce)
+				pullMethod := cdiv1.RegistryPullPod
```

Comment:
What effect does this have?

Reply:
This is to keep the negative tests running with the old method, it would be trickier to adjust them to work with the new one
(They were failing otherwise)